packages/jazz-nodejs/src/index.ts

```diff
@@ -61,5 +74,15 @@ export async function startWorker<Acc extends Account>({
     context.done();
   }
 
-  return { worker: context.account as Acc, done };
+  const inboxPublicApi = {
+    subscribe: inbox.subscribe.bind(inbox) as Inbox["subscribe"],
+  };
+
+  return {
+    worker: context.account as Acc,
+    experimental: {
+      inbox: inboxPublicApi,
+    },
```

Comment:
Wrapping the API into an "experimental" object to make it clear that it's not stable yet